### DISCUSSION TITLE
MQTT - Reaktionszeiten

### DISCUSSION THREAD

**Discussion by FMRL49** [frickelzeugs/FingerprintDoorbell#62] [Q&A]
Hi.

First of all thanks for this brilliant idea. That's the only way I've come to Home Assistant and I'm gradually getting to know the advantages.

Now to my question. Is it normal that it takes about 7 seconds for the action behind the MQTT message to be executed?

For example, if the fingerprint is recognized, it takes 7 seconds for Nuki to open the door.

**Comment by frickelzeugs** [comment]
No, for my setup the response time from "finger match" (purple led ring) to "Nuki starts opening" is usually under one second (feels almost instant). And this measurement is for the whole chain: FingerprintDoorbell -> MQTT-Broker -> OpenHAB ->Nuki Bridge -> Nuki Lock
I said "usually" because I have also delays in past, but they were related to WiFi problem at my home and occured only in rare cases.

 Do you have a Nuki 3.0 directly connected to WiFi or the old one with the separate Nuki bridge?

**Reply by FMRL49** [reply]
Thanks for the answer. Yes I connected it directly. It also takes just as long for Amazon to tell someone at the door. So it doesn't matter whether Nuki or another function is to be executed. Do I only evaluate the event via MQTT, or do I have to evaluate more? Sorry I'm a newbie.

**Reply by gsbox** [reply]
I do have the same Problem sometimes. But after a reboot of the fingerprintsensor, it works fine again .....

**Comment by FMRL49** [comment]
![Screenshot_20230116_171842_Home Assistant](https://user-images.githubusercontent.com/121131331/212724189-8f021b54-deae-45d9-9dc8-4ed4c175d25c.jpg)

**Comment by FMRL49** [comment]
Hello, topic can be closed. I did a complete reboot of my system including repeaters/routers/switches. It works now. Thanks anyway...
